kloppy/domain/models/tracking.py

```diff
@@ -24,6 +24,7 @@ class Frame(DataRecord):
     players_data: Dict[Player, PlayerData]
     other_data: Dict[str, Any]
     ball_coordinates: Point3D
+    ball_speed: Optional[int] = None
```

Comment:
Is this type hint correct? Based on the test, I'd expect a `float` instead of an `int`.

Reply:
You are correct it should be float